camunda-carbon-reductor-c8/pom.xml

```diff
@@ -54,22 +58,6 @@
             <version>${version.lombok}</version>
         </dependency>
 
-        <dependency>
-            <groupId>io.camunda.connector</groupId>
-            <artifactId>connector-validation</artifactId>
-            <version>${version.zeebe.spring-boot}</version>
-        </dependency>
-
-        <dependency>
-            <groupId>jakarta.validation</groupId>
-            <artifactId>jakarta.validation-api</artifactId>
-            <version>${version.jakarta.validation-api}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.hibernate.validator</groupId>
-            <artifactId>hibernate-validator</artifactId>
-            <version>${version.hibernate-validator}</version>
-        </dependency>
```

Comment:
These are now transitive?

Maybe you can also delete some of these ${version.[...]} variables?

Reply:
The dependencies are leftover when the Carbon Reductor was a "proper" connector. As the dependencies are no longer in use it's time to finally get rid of them.